features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher-new/source/src/app/components/Apis/Details/Monetization/index.jsx

```diff
@@ -160,8 +149,19 @@ class Monetization extends Component {
                                     />
                                 </Typography>
                                 {
-                                    (properties.length > 0) ?
-                                        (propertiesList) :
+                                    (monetizationAttributes.length > 0) ?
+                                        (monetizationAttributes.map((monetizationAttribute, i) => (
+                                            <TextField
+                                                fullWidth
+                                                id={'attribute' + i}
+                                                label={monetizationAttribute.name}
+                                                name={monetizationAttribute.displayName}
```

Comment:
should not the display name should be equivalent to label and actual name should be equivalent to name?